src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -376,4 +380,15 @@ private void sleepMax(long time, int total) {
             }
         }
     }
+
+    private void setRandomRefreshTimeIfRandomOptionActive() {
+        this.config.MISCELLANEOUS.REFRESH_TIME.VALUE = (int)(Math.random() * this.config.MISCELLANEOUS.REFRESH_TIME.RANDOM_MAX) + this.config.MISCELLANEOUS.REFRESH_TIME.RANDOM_MIN;
+        System.out.printf("Refresh time setted random as %s minutes", this.config.MISCELLANEOUS.REFRESH_TIME.VALUE);
+    }
```

Comment:
Calculate this in the refresh method. If a refresh time is not set calculate a random one, and don't set it to a config value.
Also if the current is outside the min/max you should calculate a new one (This makes it so editing the config will make a new random one be calculated too)